rest/blip_sync.go

```diff
@@ -390,6 +390,14 @@ func (bh *blipHandler) sendRevision(seq db.SequenceID, docID string, revID strin
 	outrq := blip.NewRequest()
 	outrq.SetProfile("rev")
 	seqJSON, _ := json.Marshal(seq)
+	outrq.Properties["id"] = docID
+	delete(body, "_id")
+	outrq.Properties["rev"] = revID
+	delete(body, "_rev")
+	if del, _ := body["_deleted"].(bool); del {
+		outrq.Properties["deleted"] = "1"
+		delete(body, "deleted")
+	}
```

Comment:
It doesn't need to be part of this PR, but eventually I'm hoping we'll use a version of GetRev that doesn't set these properties in the first place.  Ideally it would also return the raw bytes for the document body and use outrq.SetBody(), to avoid the JSON unmarshal-then-marshal overhead.